peps/pep-0612.rst

```diff
@@ -4,14 +4,16 @@ Author: Mark Mendoza <mendoza.mark.a@gmail.com>
 Sponsor: Guido van Rossum <guido@python.org>
 BDFL-Delegate: Guido van Rossum <guido@python.org>
 Discussions-To: typing-sig@python.org
-Status: Accepted
+Status: Final
 Type: Standards Track
 Topic: Typing
-Content-Type: text/x-rst
 Created: 18-Dec-2019
 Python-Version: 3.10
 Post-History: 18-Dec-2019, 13-Jul-2020
 
+.. canonical-typing-spec:: :ref:`typing:paramspec`
```

Comment:
Updated to match how PyPA specs are linked.

We'll need to merge https://github.com/python/typing/pull/1566 first.